playground/backend/containers/java/build.gradle

```diff
@@ -43,14 +43,6 @@ task copyDockerfileDependencies(type: Copy) {
       from 'entrypoint.sh'
       into 'build/'
    }
-   copy {
-      from '../../../infrastructure/proxy/allow_list.py'
-      into 'build/'
-   }
-   copy {
-      from '../../../infrastructure/proxy/allow_list_proxy.py'
-      into 'build/'
-   }
    copy {
```

Comment:
Why can't we use https://docs.docker.com/engine/reference/builder/#copy in the Dockerfile? Why does this need to happen in gradle?  Isn't gradle designed for building code and automating test execution? Why are we coupling gradle with deployment operations when we have these kinds of tasks as part of the docker ecosystem?

Reply:
This file is in another directory. And we just copy it to the same directory where Dockerfile is. According to the link provided by you, there is a rule:

"COPY obeys the following rules:
The <src> path must be inside the context of the build; you cannot COPY ../something /something, because the first step of a docker build is to send the context directory (and subdirectories) to the docker daemon.

If <src> is a directory, the entire contents of the directory are copied, including filesystem metadata."